DICOM/src/Microsoft.Health.DeID.SharedLib/RedactFunction.cs

```diff
@@ -0,0 +1,147 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using EnsureThat;
+using Microsoft.Health.Dicom.DeID.SharedLib.Models;
+using Microsoft.Health.Dicom.DeID.SharedLib.Settings;
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib
+{
+    public class RedactFunction
+    {
+        private static readonly int AgeThreshold = 89;
```

Comment:
❓ **QUESTION**
This value is already defined in `SharedLib.Constants` class, the problem is that it's an internal class.
Any thoughts about avoid this re-definition?

Reply:
More thoughts about this topic.
There're two constants.cs, in SharedLib and Core respectively.
And a few other values are defined in different classes like `RedactFunction `and `DicomUtility`.
These constant values can be classified to 3 categories:
1. Constants related to DICOM standard, could be used in different places. (`AgeStringLength`, `AgeThreshold`, etc.)
2. Constants defined and used by this tool. (`TagKey`, `MethodKey`, etc.)
3. Constants defined and used by a specific class. (`DateShiftSeed`, `MaxAgeValue`, etc.)

For 1, we can define them in a public (not internal) class in SharedLib.
For 2, we can defined them in an internal class in corresponding project.
For 3, we can defined them as private constant fields of the class.
Thus these values can be properly accessed.

Please note: we can discuss which constant belongs to which category. But the classifications are basically true.